### DISCUSSION TITLE
as i am using this library to extract code property graph for my code files in windows 10 but this tutorial is very confusing . would any of you elaborate it insimple steps so that i can use it in my research project as its very very urgent i request if any of you would be in touch with me to let me understand how to simply use this library

### DISCUSSION THREAD

**Discussion by anushka192001** [Fraunhofer-AISEC/cpg#866] [Q&A] [answered]
as i am using this library to extract code property graph for my code files in windows 10 but this tutorial is very confusing . would any of you elaborate it insimple steps so that i can use it in my research project as its very very urgent i request if any of you would be in touch with me to let me understand how to simply use this library

**Comment by oxisto** [comment, marked as answer]
Answered in #864
